vwifi.c

```diff
@@ -39,6 +39,12 @@ struct owl_ndev_priv_context {
     struct wireless_dev wdev;
 };
 
+static char *ssid_list = "";
+module_param(ssid_list, charp, 0644);
+MODULE_PARM_DESC(ssid_list, "Self-defined SSIDs, split by + (plus sign).");
+static const char sdssid_delim[] = "+";
+static const char default_ssid_list[] = "MyHomeWifi";
```

Comment:
Set the default SSID list via macro since it can be changed at build time.

Reply:
That is,
```c
#ifndef DEFAULT_SSID_LIST
#define DEFAULT_SSID_LIST "MyHomeWifi"
#endif
```

Reply:
fixed